### ISSUE TITLE
Lock Screen Half Swipes

### ISSUE THREAD

**Issue by seniorm0ment** [GrapheneOS/os-issue-tracker#2543]
This is actually an issue I've had for months now, so not specific to Android 14. The issue is, here and there, the lockscreen doesn't seem to want to fully accept my press/swipe to open into the numpad for the pin.

So if you click the power button to turn on the lock sceen, you see the locm icon. You can swipe on this screen, or hold your finger on the lock. Both of these should open the numpad so you can type in the pin.

The issue is with the first part, the animation to see the numpad. It does not always open when you press or swipe. It goes a little then resets. 
It gets finicky, and the animation starts and then it stops. Acting as if you took your finger off the screen, or swiped not enough.

My finger was more than well on there, and it takes me a few tries before it actually works. It acts as if my fingers are slightly wet, is another way to describe it. They are completely dry, no grease, sweat, nothing. No water on the screen, clean screen. It should have zero issues. 

I have even tried enabling increased screen sensitivity in developer options to see if it was a sensitivity issue, and it did not resolve it. 

This is not every time, but it happens often enough.

Pixel 6 Pro

**Comment by brxken128** [comment]
Do you have a screen protector on? Film/glass or anything at all? I don't really think this is a gOS issue, more of an Android design choice that was poorly thought out because it's happened to me on both stock and gOS on my Pixel 6, and stock on my Pixel 8 Pro a couple of times. It also happens on a stock Pixel 7 (not mine, but the issue isn't isolated to gOS).

It could also be a hardware issue as what you describe is a fair bit more severe than what I've ever experienced, even with a Spigen tempered glass screen protector and no increased sensitivity.

**Comment by seniorm0ment** [comment]
I do not have any screen protector on. 
Interesting, it could be the screen, or stock android. I have no way of testing it with stock android to confirm but you seem confident about that.
Hmm

**Comment by thestinger** [comment]
Try turning off developer options again as a starting point. Try using standard display size if you have that changed. If there's any accessibility service, disable that temporarily.

**Comment by seniorm0ment** [comment]
Yeah I disabled it seeing it didn't resolve it.
I was on standard display size for a while and still encountered that at the time.
And not using any accessibility services

**Comment by irelativism** [comment]
Duplicates with #2553, sorry only found this issue later. Cheers Irelativism

**Comment by seniorm0ment** [comment]
I don't think that issue is related?

**Comment by irelativism** [comment]
@seniorm0ment the issue is not the power popup menu, but the fact that you can use both interfaces can indicate there is some bug that allows grapheneOS to go back to default android one UI. And this seems to cause unresponsive touchscreen, something alike half swipes or like touchscreen cannot identify when you remove finger.

**Comment by seniorm0ment** [comment]
I have not experienced any other UI besides graphene on the lock screen, however that doesn't mean the bug isn't there which could cause this issue maybe.

**Comment by irelativism** [comment]
OK if you have a few minutes to test I would try to long press power button to reach menu and do it upwards of 10 times and see if you reproduce

**Comment by irelativism** [comment]
Sadly yet another issue closed in abusive manner so I will continue here:
"
_Lockscreen bug, 2 conflicting UIs_ #2553
Bug in lockscreen/launcher codebase is triggering a conflict between two diferent UIs. The default orange and black popup window just works while the grapheneOS one, white and blue doesn't work and freezes

Making it impossible to unlock device, only after looping through the custom lock screen UI popup window 4 times on a loop while pressing "lockdown" one is able to reach default AOSP pop-up

Here pictures respectively:
![IMG_20231015_141605](https://github.com/GrapheneOS/os-issue-tracker/assets/75645715/57b5e6d3-32a4-4548-a7bb-99db13f6b040)
![Screenshot_20231015-141436](https://github.com/GrapheneOS/os-issue-tracker/assets/75645715/d29374ac-bf2b-4ce8-bfd8-11c9bdd25b9c)

Cheers for all the work making this project a reality and time and energy reviewing this bug.
"

**Comment by thestinger** [comment]
@irelativism Last warning about being toxic.

**Comment by irelativism** [comment]
@thestinger I don't know wtf is wrong with you, but you actions are obviously abusive and hurting this project in the end

**Comment by seniorm0ment** [comment]
I was unable to replicate your issue, unless I am misunderstanding the issue or process to reproduce it.
I run into the half swipe issue still, but see no other UI pop up, or alternative UI to the power menu

**Comment by brxken128** [comment]
I started noticing this on my Pixel 6 not too long after [my initial response](https://github.com/GrapheneOS/os-issue-tracker/issues/2543#issuecomment-1763493252), and have been somewhat frequently (every couple of days) on my Pixel 8 Pro. Turning the screen off/on fixes the issue. My brother (on a Pixel 6 Pro) has also reported the issue.

I do use a Spigen AlignMaster (or whatever they're called) screen protector. However, given the amount of people that don't use one, I doubt that it's related at all.

I don't have developer options on, I don't have improved touch sensibility on, my screen is perfectly fine (across both phones), and I use no accessibility stuff. I *do* use the fingerprint sensor to unlock my profile. It definitely seems like a software thing.

I've also noticed it with the notification bar, where it just refuses to slide down fully no matter: how many times I try to drag it down, how far I drag it down, how long I try to drag it down for. It's as though the animation succeeds but doesn't trigger the next step (displaying the unlock screen or the quick settings menu). Again, turning the display off and then on again resolves the problem.

**Comment by seniorm0ment** [comment]
I have not had a screen protector on either of my devices.
On the 6 Pro maybe the case potentially because the rounded edges, however on the 8 Pro there is a visible black space between the screen and the case so it is not touching

**Comment by brxken128** [comment]
I removed the case and screen protector a few days ago, and I keep my screen clean with distilled water and a microfiber cloth - it just happened on my P8 Pro again.

**Comment by nick16** [comment]
I noticed it occurs when I swipe up starting from right beneath the lock icon.
When I swipe up from anywhere else on the screen but near the lock icon, it would not bug out.

**Comment by brxken128** [comment]
I've used the phone bare (no case, screen protector, etc) and still experience this issue. I keep the screen clean with distilled water and a microfiber cloth pretty often and I'm certain it's not hardware related.

**Comment by RA-Kooi** [comment]
I also experience this issue, both on my old Pixel 6a (destroyed) and my Pixel 7. This issue also happens when you try to pull down the top drawer. I sometimes have to pull it down over 5 times for it to finally stay down. When this issue happens both the top drawer and the lock screen have this issue.

There's also an issue, which I think might be related, where I press the flashlight or do not disturb button and the phone just vibrates shortly but nothing happens. Most of the time this happens on the lock screen, but sometimes it also happens when the device is unlocked. I haven't run into this on my Pixel 7 yet, will update when it does.
